src/components/tabController/TabBar.tsx

```diff
@@ -339,7 +349,7 @@ const TabBar = (props: Props) => {
   }, [shadowStyle, containerWidth, containerStyle]);
 
   const indicatorContainerStyle = useMemo(() => {
-    return [styles.tabBar, !_.isUndefined(height) && {height}, {backgroundColor}];
+    return [styles.tabBar, {flex: spreadItems ? 1 : undefined}, !_.isUndefined(height) && {height}, {backgroundColor}];
```

Comment:
use here a stylesheet. 
you can do `spreadItems && styles.spreadItems`

Reply:
Please see the changes here - https://github.com/wix/react-native-ui-lib/pull/1356